pom.xml

```diff
@@ -59,42 +61,58 @@
     <!-- matching armeria/grpc/zipkin -->
     <zipkin.groupId>io.zipkin.zipkin2</zipkin.groupId>
     <!-- when updating, update docker/Dockerfile -->
-    <zipkin.version>2.26.0</zipkin.version>
-    <zipkin-reporter.version>2.17.1</zipkin-reporter.version>
-    <spring-boot.version>2.7.18</spring-boot.version>
+    <zipkin.version>3.4.0</zipkin.version>
+    <zipkin-reporter.version>3.4.0</zipkin-reporter.version>
+    <spring-boot.version>3.3.0</spring-boot.version>
     <!-- armeria.groupId allows you to test feature branches with jitpack -->
     <armeria.groupId>com.linecorp.armeria</armeria.groupId>
-    <armeria.version>1.26.4</armeria.version>
-
-    <assertj.version>3.24.2</assertj.version>
-    <awaitility.version>4.2.0</awaitility.version>
-    <junit-jupiter.version>5.10.1</junit-jupiter.version>
-    <mockito.version>5.8.0</mockito.version>
+    <armeria.version>1.28.4</armeria.version>
+
+    <!-- This allows you to test feature branches with jitpack -->
+    <!--    <brave.groupId>com.github.openzipkin.brave</brave.groupId>-->
+    <!--    <brave.version>master-SNAPSHOT</brave.version>-->
+    <brave.groupId>io.zipkin.brave</brave.groupId>
+    <brave.version>6.0.3</brave.version>
+
+    <opentelemetry.version>1.39.0</opentelemetry.version>
+    <!-- Alpha jar! -->
+    <opentelemetry-proto.version>1.3.1-alpha</opentelemetry-proto.version>
+    <opentelemetry-semconv.version>1.25.0-alpha</opentelemetry-semconv.version>
+    <!-- from grpc-protobuf -->
+    <protobuf.version>3.25.1</protobuf.version>
+
+    <assertj.version>3.25.3</assertj.version>
+    <awaitility.version>4.2.1</awaitility.version>
+    <junit-jupiter.version>5.10.2</junit-jupiter.version>
```

Comment:
```suggestion
    <junit-jupiter.version>5.10.3</junit-jupiter.version>
```

Reply:
ps marcin added a different PR for this stuff, so there's a conflict. If you prefer to do a re-bump pass PR, I'll review it as he needs to rebase this anyway